_minutes/README.md

```diff
@@ -0,0 +1,23 @@
+# Meetings
+
+The [WebExtensions Community group](https://www.w3.org/community/webextensions/) meets virtually every other week, for one hour.
+Instructions to join the meeting are sent to the group's mailing list at https://lists.w3.org/Archives/Member/internal-webextensions/
```

Comment:
```suggestion
Instructions to join the meeting are sent to the group's mailing list at https://lists.w3.org/Archives/Member/internal-webextensions/2021Jun/0000.html
```

Reply:
Might be better to link to the specific email so folks don't have to go digging through the archives.

Reply:
I linked to the archive itself for now, to be consistent with your blog post at https://www.w3.org/community/webextensions/2021/06/18/wecg-public-meeting-schedule/, where you also linked to the mailing list. We were supposedly going to send an e-mail before every meeting, and perhaps we should do that in case folks aren't following our repository.

Reply:
I would have linked directly to the email if I could have, but the email wasn't ready. As I recall we wanted to get the blog post out as soon as possible, then followed up with the internal email to provide details that could not be publicly shared.

I think a regular email a few days before the meeting may be a good reminder for group members & serve as confirmation that the meeting is taking place as planned, even if it is consistent. In that case, failure to send an email would be a signal that our normal process has been interrupted. The main drawback is that this creates manual work for group organizers (even if it is relatively small edits to an existing template).

Alternatively, we could have a static doc in the repo that outlines the meeting schedule and future planned meetings. The Credentials CG does something like this in their [announcements repo](https://github.com/w3c-ccg/announcements/tree/8240293b264a6b60e6401ecb933feb8b8ac8b5b3). I use a static link here because their upcoming events table is currently empty.